pkg/lib/userexport/model.go

```diff
@@ -0,0 +1,121 @@
+package userexport
+
+import (
+	"time"
+
+	"github.com/authgear/authgear-server/pkg/api/model"
+	"github.com/authgear/authgear-server/pkg/lib/authn/authenticator"
+	"github.com/authgear/authgear-server/pkg/lib/authn/identity"
+	"github.com/authgear/authgear-server/pkg/lib/infra/redisqueue"
+	"github.com/authgear/authgear-server/pkg/util/validation"
+)
+
+// TODO: Set to 1000 as default after putting in worker task
+const BatchSize = 3
+
+type UserForExport struct {
+	model.User
+
+	Identities     []*identity.Info
+	Authenticators []*authenticator.Info
+}
+
+type Error struct {
+	Message string `json:"message,omitempty"`
+	Reason  string `json:"reason,omitempty"`
+}
```

Comment:
Remove this. You should use apierrors.Error

Reply:
removed